### ISSUE TITLE
App signin from bog post does not remember context of post URL

### ISSUE THREAD

**Issue by CameraCornet** [symfony/demo#763]
When a reader wishes to comment on a blog post, and is not signed in, the app does not remember which blog post they were reading.

For example, if you are on:
http://localhost:8000/en/blog/posts/lorem-ipsum-dolor-sit-amet-consectetur-adipiscing-elit

And you sign-in to comment, it takes you back to the http://localhost:8000/en/blog/ listing - and not back to the specific post you hit 'sign in' from.

**Comment by yceruto** [comment]
You're right and should be fixed in https://github.com/symfony/demo/pull/764
